Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.12-alpine
+
+LABEL maintainer="Karthic Rao<karthic@dgraph.io>"
+
+ENV GOPATH /go
+ENV GO111MODULE on
+
+WORKDIR /app
+
+RUN  \
+     apk add --no-cache git && \
+     git clone https://github.com/dgraph-io/flock.git && cd flock && \
+     go install -v
+
+ENTRYPOINT ["/go/bin/flock"]
```

Comment:
Avoid entrypoint. Just use CMD only

Reply:
Any specific reason? Added the entrypoint since the purpose of the container was to just execute flock by default.

Reply:
Well, we had problems debugging the image because of entrypoint.